frontend/src/components/layout/feature-panel/FeaturePanel.tsx

```diff
@@ -27,7 +27,7 @@ const Subtitle = styled.h2`
 
 export const FeaturePanel: React.FC = (): JSX.Element => {
```

Comment:
No necesitas tipar la salida ya que TS lo infiere:
```suggestion
export const FeaturePanel: React.FC = () => {
```